index.html

```diff
@@ -240,45 +281,59 @@ <h2>About me</h2>
          so if you'd like to chat please get in touch.
       </p>
     </div>
-   <div>
+   <div class="mobile-form">
     <form action="https://formspree.io/f/mwkzggnv" method="post">
       <input type="text" placeholder="Full name" maxlength="30" id="text" name="text" required>
       <input type="email" id="email" name="email" placeholder="Email address" maxlength="30" required>
       <textarea name="" id="" cols="10" rows="5" required maxlength="500" name="textarea" id="textarea"  placeholder="Enter text here"></textarea>
       <button class="Multi-Post-project button form-button" type="submit">Get in touch</button>
     </form>
    </div>
+    <div class="deskstop-form">
+      <form action="https://formspree.io/f/mwkzggnv" method="post">
+        <div class="input-form">
+          <input type="text" placeholder="First Name" class="first-name" maxlength="30" id="text" name="text" required>
+        <input type="text" placeholder="Last Name" maxlength="30" id="text" name="text" required>
+        </div>
```

Comment:
- [ ] Based on the requirement, you should use one text input field for the name part of the form. Kindly implement this. 👍 The single input field that you will be using should take the space of both First Name and Last Name inputs. 💻